src/Identity/Swauth/Service.php

```diff
@@ -2,14 +2,11 @@
 
 namespace Recras\OpenStack\Identity\Swauth;
 
-use Recras\OpenStack\Identity\Swauth\Models\Token;
-use Recras\OpenStack\Identity\Swauth\Models\ServiceCatalog;
-
 use GuzzleHttp\ClientInterface;
-use OpenStack\Common\Service\AbstractService;
 use OpenStack\Common\Auth\IdentityService;
-use OpenStack\Common\Api\Operation;
-use OpenStack\Common\Api\ApiInterface;
+use OpenStack\Common\Service\AbstractService;
+use Recras\OpenStack\Identity\Swauth\Models\ServiceCatalog;
+use Recras\OpenStack\Identity\Swauth\Models\Token;
```

Comment:
consider to alias these `use`s, so this use-context is not necessary for reading the method implementation:
`use Recras\OpenStack\Identity\Swauth\Models\Token as RecrasToken;`
(or skip the `use` statement and use partially namespaced class names, i.e. `Models\Token::class` because this file is in the `Recras\OpenStack\Identity\Swauth` namespace)

Reply:
It just occured to me that `SwauthToken` would be a far better namespace alias than `RecrasToken`

Reply:
The coding convention from the base Openstack library seems to prefer `use` statements without aliases. For example see `\OpenStack\Identity\v3\Service`.